regen_doc.g

```diff
@@ -1,6 +1,4 @@
 # Regenerate parts of the recog documentation from its source code
-
-SetPackagePath("recog", "./");
```

Comment:
The problem with that is that we can easily end up loading the "wrong" recog, and then we also generate writing wrong documentation.

Reply:
Ah, I thought it was not that easy to load the wrong recog. I inferred from your other comments that AutoDoc would complain if an older version was loaded. Thus I thought this could only be problematic for people that have two different versions of say the development version. And I hope this situating does not occur when compiling the documentation of a release. Or am I wrong here? 

I usually use the `makedoc.g` to compile the documentation of any manual. So I thought it would be a good idea to tell people that they should call `gap -A makedoc.g`. The call `gap makedoc.gʼ will not succeed unless one removes the recog release from the `GAPROOT/pkg` directory.

Reply:
I somehow missed your comment in #64. What you said here sounds good to me:
> What could be done, though, is this: if recog is not loaded, use SetPackagePath to load the correct version. If it is loaded, inspect its package info record, and check its InstallationPath to verify that it refers to the correct directory.

Reply:
Is there some way to get the absolute path that a directory object, e.g.  `Directory("./")`, represents?

Reply:
You can use `IO_getcwd` from the IO package to get the path of the current directory; this was added in IO version 4.5.4 (2018-09-19) specifically for that purpose.